src/node/signer.ts

```diff
@@ -43,22 +48,30 @@ export class TurboNodeArweaveSigner implements TurboWalletSigner {
   async signDataItem({
     fileStreamFactory,
     fileSizeFactory,
+    opts = {},
```

Comment:
Is there a material difference in signatures when you pass an empty object here vs. undefined?

Reply:
A quick inspection of arbundles indicates that it would be fine, but if it's not necessary for the object to exist in the remaining body of this function then I'd just make it an optional parameter.

Reply:
1dd5929